lua/luasnip/extras/fmt.lua

```diff
@@ -219,4 +228,5 @@ return {
 	-- alias
 	fmt = format_nodes,
 	fmta = extend_decorator.apply(format_nodes, { delimiters = "<>" }),
+	fmtrp = extend_decorator.apply(format_nodes, { repeat_duplicates = true }),
```

Comment:
Ah, I'm a bit on the fence with this, normally `extend_decorator` should be used locally to generate such "overloads".
(fmta is an exception because it existed before extend_decorator, and there's not much reason to remove it)

Do you have some arguments in favor of providing this overload here?

Reply:
Not really, I just think it’s nice to have a shortcut for this, I guess I can add this in my local config also.

Reply:
I think I'd prefer that, it's easy to add them, and if we add more here it'll be hard to draw a line in the future

Reply:
yup

Reply:
Ty😊